src/lib/components/event/payload-decoder.svelte

```diff
@@ -54,12 +54,11 @@
         convertedAttributes,
       ) as object;
       const keyExists = key && decodedAttributes?.[key];
-      if (keyExists) {
-        decodedValue = stringifyWithBigInt(keyExists);
-      } else {
-        decodedValue = stringifyWithBigInt(decodedAttributes);
+      let finalValue = keyExists ? decodedAttributes[key] : convertedAttributes;
```

Comment:
Should `convertedAttributes` be `decodedAttributes` here like it was previously? Or was the change intentional? 

```suggestion
      let finalValue = keyExists ? decodedAttributes[key] : decodedAttributes;
```